cfpb/cfgov-refresh
# Convert Bureau Structure to Wagtail

Convert Bureau Structure to Wagtail

## Additions

- Added CSS file to customize Wagtail Admin for the Bureau Structure.

## Changes

- Modified Jinja, Less, and python files to migrate the bureau structure page from a Sheer page to Wagtail. I left the current Sheer Page in place and just updated the necessary associated files to make it atomic.

## Testing
- Run `gulp build`
- Add a Browse page via Wagtail.
- Add a Bureau Structure block to the page
- Verify that the page previews as expected. 

## Review

- @cfpb/cfgov-backends 
- @Scotchester 

## Screenshots


## Notes

- It might make sense to use snippets at some point but it would have to more flexible then just injecting that snippet into a 25 / 75 organism

- The CSS modifications to the Wagtail admin are experimental and will probably need to be adjusted.

## Todos

-

## Checklist
